.buildkite/pipeline.yml

```diff
@@ -9,6 +9,10 @@ steps:
     label: Executable
     artifact_paths:
       - "result/bin/xrefcheck"
+  - command: nix-build --argstr platform windows
```

Comment:
It looks like an error happens during build, but it does not fail the build: https://buildkite.com/serokell/xrefcheck/builds/284#c03312be-8e80-4bcc-baa4-84505668ae18/65-144. Do you know if this error is critical?